ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -65,6 +63,24 @@ export class Burn extends IronfishCommand {
       account = defaultAccount.name
     }
 
+    let assetId = flags.assetId
+    if (!assetId) {
+      assetId = await CliUx.ux.prompt('Enter the Asset Identifier to burn supply for', {
+        required: true,
+      })
+    }
+
+    let amount
+    if (flags.amount) {
+      amount = CurrencyUtils.decodeIron(flags.amount)
+    } else {
+      const input = await CliUx.ux.prompt('Enter the amount to burn in $IRON', {
```

Comment:
$IRON is Iron Fish's asset, IRON is a denomination.

```suggestion
      const input = await CliUx.ux.prompt('Enter the amount to burn in IRON', {
```